drivers/usb/usbstor/usbstor.c

```diff
@@ -192,11 +192,8 @@ DriverEntry(
 
     DPRINT("********* USB Storage *********\n");
 
-    DriverObject->DriverUnload = USBSTOR_Unload;
-    DriverObject->DriverExtension->AddDevice = USBSTOR_AddDevice;
-    DriverObject->DriverStartIo = USBSTOR_StartIo;
-    DriverObject->MajorFunction[IRP_MJ_CREATE] = USBSTOR_DispatchClose;
-    DriverObject->MajorFunction[IRP_MJ_CLOSE] = USBSTOR_DispatchClose;
+     if(1)
+         DriverObject->DriverUnload = USBSTOR_Unload; DriverObject->DriverExtension->AddDevice = USBSTOR_AddDevice; DriverObject->DriverStartIo = USBSTOR_StartIo; DriverObject->MajorFunction[IRP_MJ_CREATE] = USBSTOR_DispatchClose; DriverObject->MajorFunction[IRP_MJ_CLOSE] = USBSTOR_DispatchClose;
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
 Lines should be <= 50 characters long  [whitespace/line_length] [2]

Reply:
🐶🐶